### PR TITLE
Bump version to install several net tools in docker

### PR DESCRIPTION
**Pull request by StevenJiang1110**
Such tools are required by #1480 and #1482.

`grup-bc-bin` for supporting IOMMU is only included in this PR.

### CODE REVIEW

File: tools/docker/Dockerfile.jinja [hunk 1]
```diff
@@ -374,6 +374,7 @@ RUN apt update && apt-get install -y --no-install-recommends \
     grub-efi-amd64-dbg \ 
     ovmf                `# provide an alternative stable firmware` \ 
     {% endif %}
+    grub-pc-bin         `# for IOMMU support` \
```
[the remark this answers is not in the document]
Comment (lucassong-mh) [reply tools/docker/Dockerfile.jinja:377]: Got it...The initial attempt is to make enabling iommu on x86_64&ubuntu workable for now. Simply `make run SCHEME=iommu` still don't work if we're not changing grub.

[the remark this answers is not in the document]
Comment (tatetian) [reply tools/docker/Dockerfile.jinja:377]: So why IOMMU cannot be supported if `grub-efi-amd64` is used?

**Review by tatetian — APPROVED** [review]
LGTM. Thanks for the contribution!

File: tools/docker/Dockerfile.jinja (hunk 1 above)
[the remark this answers is not in the document]
Comment (junyang-zh) [reply tools/docker/Dockerfile.jinja]: I don't have a clear answer. Haven't investigated it for quite a while since.

**Pull request merged**